### PR TITLE
Fix use of deprecated _async_send method in pyppeteer

### PR DESCRIPTION
**Pull request by itsdalmo**
With 3.9 the `_async_send` method has been deprecated and causes AADA to print errors in a loop. The fix is to bump the pyppeteer version used by AADA, and its dependencies. Since the API for pyppeteer has changed we also need to make code changes to AADA - and it seems like we can just use the pyppeteer Launcher directly (without subclassing it in AADA).

This change works on my computer™️  
Let me know what you think (I don't really know Python, but this was bugging me so I was hoping to fix it 😅 )

### CODE REVIEW

**Comment by nierob** [comment]
I have verified that it works on my computer with python 3.9 too. Looks good to me!

File: aada/launcher.py [hunk 1]
```diff
@@ -11,68 +11,5 @@
...
-            if not check_chromium():
-                download_chromium()
-            self.exec = str(chromium_executable())
-
-        self.cmd = [self.exec] + self.chrome_args
-
-
 async def launch(options: dict = None, **kwargs: Any) -> Browser:
-    return await HeadLessLauncher(options, **kwargs).launch()
+    return await Launcher(options, **kwargs).launch()
```
Comment (nierob) [inline aada/launcher.py]: Probably you can remove the whole file and just use `Launcher(options, **kwargs).launch()` everywhere.

**Comment by denisbr** [comment]
@itsdalmo Great! Just a quick question, did you test if it was working with Keyring? I haven't had the chance to test myself yet, although I did encounter the same issue as you with py3.9.

File: aada/launcher.py (hunk 1 above)
Comment (itsdalmo) [reply aada/launcher.py]: Fixed it in the latest commit (amended and force pushed since I probably should have done that to begin with) 👍

**Comment by itsdalmo** [comment]
> Great! Just a quick question, did you test if it was working with Keyring?

It seems to be doing lookups in keyring without issue on my end @denisbr (at least I'm not seeing any errors)

**Comment by itsdalmo** [comment]
Any chance of getting this reviewed @piontas? 😅

**Review by piontas — APPROVED** [review]
LGTM. Thank you!

**Comment by piontas** [comment]
> Any chance of getting this reviewed @piontas? 😅

Approved and merged! Thank you for your contribution!

**Pull request merged**